<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;
font: 11pt Poppins;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="backBtn">
    <property name="geometry">
     <rect>
      <x>435</x>
      <y>0</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	height: 40px;
	border-radius: 20px;
	border: 1px solid #00007f;
	background-color: white;
	color: #00007f;
	font-weight: 500;
	font-size: 16px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widgetKeyboard" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>480</width>
      <height>135</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(234, 234, 234);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveBtn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color: #00007f;
color: white;
border: none;
border-radius: 15px;
font-size: 11pt;
</string>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Valor 1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Valor 2&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Valor 3&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Valor 4&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSPinVal1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>101</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);
border-style: solid;
border-width: 1px;
border-radius: 6px;
font: 12pt &quot;Poppins&quot;;
background-color: rgb(234, 234, 234);</string>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="suffix">
     <string/>
    </property>
    <property name="maximum">
     <double>500.000000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSPinVal2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>101</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);
border-style: solid;
border-width: 1px;
border-radius: 6px;
font: 12pt &quot;Poppins&quot;;
background-color: rgb(234, 234, 234);</string>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="maximum">
     <double>500.000000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSPinVal3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>101</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);
border-style: solid;
border-width: 1px;
border-radius: 6px;
font: 12pt &quot;Poppins&quot;;
background-color: rgb(234, 234, 234);</string>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="maximum">
     <double>500.000000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSPinVal4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>101</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);
border-style: solid;
border-width: 1px;
border-radius: 6px;
font: 12pt &quot;Poppins&quot;;
background-color: rgb(234, 234, 234);</string>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="showGroupSeparator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <double>500.000000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
